src/main/java/org/medibloc/panacea/domain/Transactions.java

```diff
@@ -115,13 +93,6 @@ public static Tx createTxWithSign(List<Wallet> wallets, TxBody txBody, AuthInfo
                 .build();
     }
 
-    public static Tx createTxWithSign(List<Wallet> wallets,
-                                      GeneratedMessageV3 msg,
-                                      String memo,
-                                      Fee fee) throws IOException, NoSuchAlgorithmException {
-        return createTxWithSign(wallets, Collections.singletonList(msg), memo, fee);
-    }
```

Comment:
Wouldn't it be better to leave this method as it is possible to use only one msg?

Or, I think delete it if don't use it outside, but I might need a guide if need.

Reply:
Ja. I just reverted all removed methods. Please review it again.